skill/lambda/index.js

```diff
@@ -338,14 +334,18 @@ const ShowBadgesIntentHandler = {
         const prompt = "What would you like to do?";
         speakOutput += prompt;
         
-        handlerInput.responseBuilder.addDirective({
-            "type":"Alexa.Presentation.HTML.HandleMessage",
-            "message": {
-                "intent":"showBadges",
-                "playAnimation": true,
-                "gameState": getProfile(handlerInput)
-            }
-        });
+        
+        // TODO: Ask Joe why he's calling getProfile when we already have profile.
```

Comment:
Probably because I did not notice it above :) You can remove the second call for a reference to the above and delete this comment

Reply:
Going to merge this in and fix it.